content-src/asrouter/templates/Trailhead/Trailhead.jsx

```diff
@@ -70,15 +128,25 @@ export class Trailhead extends React.PureComponent {
             <a data-l10n-name="benefits"
               href="#" />
           </p>
-          <input type="email" data-l10n-id="onboarding-join-form-email" />
-          <p data-l10n-id="onboarding-join-form-email-error" />
-          <p data-l10n-id="onboarding-join-form-legal">
-            <a data-l10n-name="terms"
-              href="https://accounts.firefox.com/legal/terms" />
-            <a data-l10n-name="privacy"
-              href="https://accounts.firefox.com/legal/privacy" />
-          </p>
-          <button data-l10n-id="onboarding-join-form-continue" />
+          <form method="get" action={this.props.fxaEndpoint} target="_blank" rel="noopener noreferrer" onSubmit={this.onSubmit}>
+            <input name="service" type="hidden" value="sync" />
+            <input name="action" type="hidden" value="email" />
+            <input name="context" type="hidden" value="fx_desktop_v3" />
+            <input name="entrypoint" type="hidden" value="activity-stream-firstrun" />
+            <input name="utm_source" type="hidden" value="activity-stream" />
+            <input name="utm_campaign" type="hidden" value="firstrun" />
```

Comment:
I wonder if the FXA team wants new values for any of these params.

Reply:
Per the copy doc, I added utm_term=trailhead so the FxA folks can filter the data easily.